packages/quantic/force-app/main/default/lwc/quanticSearchBox/quanticSearchBox.js

```diff
@@ -35,22 +43,39 @@ export default class QuanticSearchBox extends LightningElement {
   input;
   /** @type {import("lwc").HTMLElementTheGoodPart} */
   combobox;
-
+  /** @type {HTMLButtonElement} */
+  clearButton;
+  /** @type {HTMLElement} */
+  closeButton
```

Comment:
What is a closeButton exactly?

Reply:
The mockups have two different icons for the `X` button to clear the input. One is called clear and the other is called close, that's why I called one clearButton and the other closeButton. They do the same thing but I couldn't give them the same Id and use only one element for both.

Reply:
They should never both appear at the same time, can't it just be one reference to `clearButton`, the only thing that changes about it is the name of the icon used right?

Reply:
Actually pay no attention to my argument ^, it was nonsense. It works using `clearButton` for both elements